specification/postgresqlhsc/resource-manager/Microsoft.DBforPostgreSQL/preview/2023-03-02-preview/postgresqlhsc.json

```diff
@@ -2322,6 +2334,30 @@
         }
       }
     },
+    "DataEncryption": {
+      "type": "object",
+      "description": "The data encryption properties of a cluster.",
+      "properties": {
+        "primaryKeyUri": {
+          "type": "string"
+        },
+        "primaryUserAssignedIdentityId": {
+          "type": "string",
+          "description": "Resource Id for the User assigned identity to be used for data encryption of the primary server."
+        },
+        "type": {
+          "type": "string",
+          "enum": [
+            "AzureKeyVault",
+            "SystemAssigned"
```

Comment:
since the property name is `primaryUserAssignedIdentityId`, why enum name is `SystemAssigned`? Shouldn't it be `UserAssigned` or `ManagedIdentity` ?